content/en/docs/instrumentation/java/automatic/withspan.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Use instrumentation annotations
```

Comment:
How about simply "Annotations" for the title, because this page is already under "Instrumentation".